ONSdigital/design-system
# Fix dsExample param on error panels

### What is the context of this PR?
This is to fix the logic around the `dsExample` param on error panels. When set to true it is designed to not autofocus on the error panels when used in the Design System examples. It was no longer stopping the autofocus, this will fix that.

### How to review 
- Check that on the panels page the error panels aren't autofocused
- Check to see that they are autofocused if the `dsExample` param is removed
